TestClient/MainWindow.cs

```diff
@@ -294,10 +294,11 @@ private void btnListScenes_Click(object sender, EventArgs e)
                 foreach (var scene in scenes)
                 {
                     var node = new TreeNode(scene.Name);
-                    foreach (var item in scene.Items)
-                    {
-                        node.Nodes.Add(item.SourceName);
-                    }
+                    // TODO
```

Comment:
Explain what "Todo" i.e. // TODO Uncomment towards v5 release

Reply:
Was already addressed & removed in subsequent commits